packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -33,17 +35,14 @@ let uid = 0;
 export default {
     name: 'VLink',
 
+    inheritAttrs: false,
+
     props: {
         isExternalSite: {
             type: Boolean,
             default: false
         },
 
-        isRouterLink: {
```

Comment:
What replaces `isRouterLink`? We have a case where that needs to be set.

Reply:
@zlatinivanov-je If you pass in the `to` attribute instead of `href`, it will behave as a router-link.

Reply:
@zlatinivanov-je This logic was actually changed in 2.2.0 but the prop is leftover. The `linkType` computed property determines if to render `<a>` or `<router-link>` based on the `to`/`href` attributes.